articles/native-platforms/ionic/08-mfa.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Multifactor Authentication
+description: This tutorial will show you how to add Multifactor Authentication to your Ionic app with auth0.
+---
+
+<%= include('../../_includes/_package', {
+  githubUrl: 'https://github.com/auth0-samples/auth0-ionic-samples',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-ionic-samples',
+  pkgBranch: 'master',
+  pkgPath: '08-MFA',
+  pkgFilePath: null,
+  pkgType: 'replace'
+}) %>
+
+<%= include('../_includes/_mfa-introduction') %>
```

Comment:
@lobo Throws `Error: Could not find include ../_includes/_mfa-introduction`. There is no such directory.